modules/ROOT/pages/dataweave-formats-multipart.adoc

```diff
@@ -426,6 +427,54 @@ Content-Disposition: form-data; name="otherJsonPart"
 ------myboundary------
 ----
 
+[[example4]]
+=== Example: Iterate through Multipart payload
+
+This example shows how DataWeave iterates through through all the parts of the payload and process the message individually.
```

Comment:
```suggestion
This example shows how DataWeave iterates through all the parts of the payload and processes the each part individually.
```

Reply:
"and processes the message individually" is unclear. I assume it means "each part of the message"? I'm not sure but that's what I suggested above.